HackerNewsReader/Classes/AppDelegate.m

```diff
@@ -36,6 +38,12 @@ - (void)setupAppirater {
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     [self setupAppirater];
 
+    // Basic integration with Sift.
+    Sift *sift = [Sift sharedInstance];
+    [sift setAccountId:@"YOUR_ACCOUNT_ID"];
+    [sift setBeaconKey:@"YOUR_JAVASCRIPT_SNIPPET_KEY"];
+    [sift setAllowUsingMotionSensors:YES];
```

Comment:
I updated there to make sure that the style here matches the integration guide (README.md).